Vagrantfile.example

```diff
@@ -60,6 +60,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     dev.vm.provider :libvirt do |domain, override|
         domain.cpus = 4
+	# On some hosts, the guest gets stuck at "Waiting for domain to get an IP address..." if
+	# the default cpu_mode, 'host-model', is used. Due to known issues, this mode is not
```

Comment:
As a middle sentence, can you mention that host-passthrough prevents VM migration, but we do not expect developer VMs to be migrated between hypervisors?